test/unit/filters/realtiveTimeFilterSpec.js

```diff
@@ -0,0 +1,60 @@
+/* global describe, it, inject, expect, beforeEach, jasmine, tsNow*/
+
+describe('relativeTime filter', function () {
+  var $filter, _, relativeTimeFilter, minuteSpy, hourSpy
+
+  beforeEach(module('myApp.filters'))
+
+  beforeEach(inject(function (_$filter_, ___) {
+    $filter = _$filter_
+    _ = ___
+  }))
+
+  beforeEach(function () {
+    minuteSpy = jasmine.createSpy()
+    hourSpy = jasmine.createSpy()
+    _.pluralize = function (msgid) {
```

Comment:
in the filter the difference between the input timestamp and the current timestamp is used by the functions ``_.pluralize('relative_time_pluralize_minutes_ago')`` and ``_.pluralize('relative_time_pluralize_hours_ago')``.
Because the difference is hard to estimate, I replaced the pluralize function to return Jasmine spies, so it can at least be checked if they were called.